Ladybird/Tab.cpp

```diff
@@ -521,6 +521,14 @@ void Tab::location_edit_return_pressed()
     navigate(m_location_edit->text());
 }
 
+void Tab::open_file()
+{
+    m_open_file = QFileDialog::getOpenFileName(this, tr("Open file"), QDir::homePath(), tr("All Files (*.*)"));
+
+    if (!m_open_file.isEmpty()&& !m_open_file.isNull())
+        navigate("file://"+m_open_file);
```

Comment:
clang-format is angry here in the CI about missing spaces

```suggestion
    if (!m_open_file.isEmpty() && !m_open_file.isNull())
        navigate("file://" + m_open_file);
```

Reply:
This is so confusing, clion loaded .clang-tidy that told me to remove the spaces(I think). 😳